src/test/java/ladder/LineTest.java

```diff
@@ -8,47 +8,53 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import ladder.domain.Cross;
 import ladder.domain.Line;
+import ladder.domain.Point;
 
 public class LineTest {
 
 	@Test
 	@DisplayName("라인 생성 테스트")
 	void lineMakeTest() {
 		Line line = new Line(countOfPerson -> {
-			List<Boolean> points = new ArrayList<>();
-			points.add(false);
-			points.add(true);
-			points.add(false);
-			points.add(false);
-			return points;
+			Point first = Point.first(false);
+			List<Cross> crosses = new ArrayList<>();
+			crosses.add(new Cross(0, first));
+			crosses.add(new Cross(1, first.next(true)));
+			crosses.add(new Cross(2, first.next(true).next(false)));
+			crosses.add(new Cross(3, first.next(true).next(false).last()));
+			return crosses;
 		}, 4);
-		assertThat(line.getPoints().size()).isEqualTo(4);
-		assertThat(line.getPoints()).containsExactly(false, true, false, false);
+		assertThat(line.getCrosses().size()).isEqualTo(4);
 	}
 
 	@Test
 	@DisplayName("라인 생성 랜덤 테스트")
 	void lineMakeRandomTest() {
 		Line line = new Line(5);
-		assertThat(line.getPoints().size()).isEqualTo(5);
+		assertThat(line.getCrosses().size()).isEqualTo(5);
```

Comment:
다음과 같이 hasSize 를 사용해볼 수 있을 것 같아요! 😄 
```suggestion
		assertThat(line.getCrosses()).hasSize(5);
```